back_ends/bash/exec.sh

```diff
@@ -1,8 +1,22 @@
 code_file_path="$1"
 test_code_file_path="$2"
+check_code_file_path="$3"
 # We won't use this because this back end only supports text-based back ends.
 #output_type="$3"
 
+if [ -f "$check_code_file_path" ]
+then
+  # adds a newline to student's code if not already present, ensures that bash
+  # checking code doesn't skip the final line
+  sed -i -e '$a\' "$code_file_path"
+  value=`bash $check_code_file_path`
```

Comment:
Wondering if you need to surround with double quotes?

value="`bash $check_code_file_path`"

if [[ "$value" ]]

Reply:
```if [[ "$value" ]]``` works! 
```value="bash $check_code_file_path"``` sets value to a string rather than the command's output but we could also do ```value=$(bash $check_code_file_path)```, and that's probably more intuitive/in line with our styling than the template literals. I changed it to use the $() syntax!